package/MDAnalysis/analysis/distances.py

```diff
@@ -55,6 +54,9 @@ def contact_matrix(coord, cutoff=15.0, returntype="numpy", box=None):
     .. versionchanged:: 0.11.0
        Keyword *suppress_progmet* and *progress_meter_freq* were removed.
     '''
+    
+    from scipy import sparse      
```

Comment:
It doesn't really make sense here though.  If we put the import at the top, then the entire module is inaccessible to non scipy people, whilst it looks like there's only one option in one function that needs it.